### PR TITLE
Refactoring init.js of Designer

### PR DESCRIPTION
**Pull request by Bournvita1998**
Signed-off-by: Bournvita1998 <mohit.kuri@research.iiit.ac.in>

### Description

Refactoring in the init.js of the designer section

Fixes #

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

### CODE REVIEW

**Review by shucon — APPROVED** [review]

**Review by williamdes — APPROVED** [review]

**Comment by ibennetch** [comment]
Good job.

File: js/designer/init.js [hunk 1]
```diff
@@ -54,22 +54,22 @@ AJAX.registerOnload('designer/init.js', function () {
     }
 
     $('#query_Aggregate_Button').on('click', function () {
-        document.getElementById('query_Aggregate').style.display = 'none';
+        $('#query_Aggregate').style.display = 'none';
```
Comment (devenbansod) [inline js/designer/init.js]: @Bournvita1998 shouldn't we be using `$("#id").css("display", "none");` or `$("#id").hide();` here?

Comment (Bournvita1998) [reply js/designer/init.js]: @devenbansod I didn't find a good article comparing the difference between these 3. I know they all does the same job but no idea what's the difference. Can you help me knowing the difference between them?

Comment (Bournvita1998) [reply js/designer/init.js]: @devenbansod, for now, I am thinking of moving ahead making the changes in a similar way as I have done here. If required, we will change them again.

Comment (devenbansod) [reply js/designer/init.js]: @Bournvita1998 have you tested this in the browser?

AFAIK, `$('selector')` returns an array-like JQuery-native object and doesn't return a native DOM element. So if I'm not mistaken, this code change would throw an error: `Uncaught TypeError: Cannot set property 'display' of undefined` in the console.

Ref for the `css` API: https://api.jquery.com/css/

Comment (Bournvita1998) [reply js/designer/init.js]: @devenbansod Yeah you are right here. $('selector') will return an array of all possible matches but document.getElementById('ID') returns just the first match it finds. To resolve this error we can either do the things as you suggested in your first comment or we can change $('selector') to $('selector')[0]. I found [this](https://stackoverflow.com/questions/6974582/jquery-object-and-dom-element) on stackoverflow explaining the difference between jQuery object and DOM elements.

Also, I found the same issue in my previously created PRs too, which need to be changed soon. I am thinking of replacing them using the css format like:  $("#id").css("display", "none");  is that fine to move ahead?

Comment (devenbansod) [reply js/designer/init.js]: Sure, please go ahead 👍

Comment (williamdes) [reply js/designer/init.js:57]: This was indeed invalid: https://github.com/phpmyadmin/phpmyadmin/issues/16960

**Comment by williamdes** [comment]
Gosh, why did I approve this invalid code.. 🤦🏻 
Ref: #16960

**Pull request merged**